src/components/ProfileButton/ProfileButton.jsx

```diff
@@ -5,26 +5,34 @@ import defaultIcon from "../../assets/icons/person.svg";
 import styles from "./ProfileButton.module.css";
 
 const ProfileButton = ({
-  children,
-  onClick,
+  size = "small",
+  src = defaultIcon,
   className,
+  onClick,
+  children,
   disabled = false,
-  src = defaultIcon,
 }) => (
   <button
-    className={clsx(styles.profileButton, className)}
-    disabled={disabled}
+    className={clsx(
+      styles.profileButton,
+      size === "large" ? styles.large : styles.small,
```

Comment:
아래처럼 작성하면 더 효율적일 것 같아요 😊.
(이후 사이즈에 대한 클래스가 더 추가되어도 이 코드는 수정 안해도 될 것 같아요.)
```suggestion
    styles[size],
```